worker.go

```diff
@@ -268,7 +268,7 @@ func doWork(
 	defer func() {
 		if r := recover(); r != nil {
 			debug.PrintStack()
-			problems.Add("panic caused by: %w", r)
+			problems.Add("panic caused by: %v", r)
```

Comment:
hmm that change is weird

Reply:
This is the error:

```
./worker.go:271: (*github.com/zalando-incubator/kube-ingress-aws-controller/problem.List).Add format %w has arg r of wrong type interface{}
```

Would you propose some other formatting?

Reply:
hmm, I think we did a mistake in the func definition of problem.Add. I think we should have an error passed not a format string nor empty interface list.

Reply:
So let's keep your %v change and refactor it later.

Reply:
Maybe we may cast `recover()` result to `error`https://pkg.go.dev/builtin#recover

Reply:
@szuecs pointed out that `recover` result type may be anything so `%v` seems to be the right way.